### ISSUE TITLE
Unable to restore username/cannot create a new id

### ISSUE THREAD

**Issue by ryanarndtcm** [blockstack/blockstack-browser#2008]
This is coming in from a user in support who has been sent a link to this issue to add to.

_After creating my account I cleared the browser which prompted me to recover my account. I was able to recover my account but my username has not been recovered. I cannot create another id, please advise._

**Comment by markmhx** [comment]
Their ID may not be showing up because a certain amount of time post-registration needs to pass for paid IDs. We may want to ask them to check again now that it's been several days.

If it still doesn't show up, I wonder if selecting "Add another ID" under Identity might help restore it (although I'm not clear as to whether this is generally necessary for the _first_ ID they've registered or just subsequent ones cc. @hstove ?)

**Comment by markmhx** [comment]
@ryanarndtcm have we been able to confirm the user just needed to wait here?

**Comment by ryanarndtcm** [comment]
there has been no response from the end user. reaching out again now.

**Comment by maxkleinke** [comment]
this also happens when you register
e.g. I registered with the it "helloitsme.id.blockstack", put in a mail that does not exist (if that matters), got my recovery credentials and instead of displaying "helloitsme.id.blockstack" after the registration I'm getting "ID-something"

**Comment by markmhx** [comment]
@maxkleinke mind indicating which app you were using so we can try recreating the problem with it?

**Comment by maxkleinke** [comment]
@markmhx its a private university project the link we were using is https://browser.blockstack.org/sign-up?redirect=%2Fauth%3FauthRequest%3D and some jwt token i believe. However i wasnt able to reproduce it myself, will let you know if i figure it out

**Comment by trentguillory** [comment]
If I navigate to browser.blockstack.org, create an account, log out, and immediately try logging back in with my secret key, it displays "ID-something" instead of the ID I just created. I've tried several times - it doesn't ever allow re-logging in for me.

I figured this info might help since there's no app involved. Just Blockstack.

**Comment by ryanarndtcm** [comment]
That may be due to the new ID not being registered on the blockchain quite yet - if you use that same ID 24 hours later, does it continue to not show up?

**Comment by trentguillory** [comment]
@ryanarndtcm you're right - just waiting a few hours resolved the issue. I didn't realize that was the norm, thanks for helping out!

**Comment by hstove** [comment]
The issue is due to how we "poll" for the username information. It's a bit wonky in the Browser, I apologize for that. Usually, if a username isn't showing up, you can click the "Profile" tab. This will force refresh all usernames. It should show up after doing that step.
